torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -3535,11 +3535,6 @@ Error PyTorchModelLoader::loadBatchNorm(const torch::jit::Node *ptNode) {
                                            inputs[BatchNormInputs::training])));
   RETURN_ERR_IF_NOT(training == false, "Don't support BatchNorm training yet.");
 
-  RETURN_ERR_IF_NOT(
-      input.dims().size() == numDims + 2,
-      glow::strFormat("Number input dimensions must be equal to %d, got %lu",
-                      numDims + 2, input.dims().size()));
-
   size_t numChannels = input.dims()[1];
```

Comment:
Here we are relying on the knowledge of "second dim is channel." If input.dims.size() != numDims + 2, does this still satisfy?